packages/api/src/app.ts

```diff
@@ -82,7 +92,23 @@ const app: FastifyPluginAsync<AppOptions> = async (
     next()
   }
 
+  // InitializeModels and callback
+  const initializeTradeCooldowns: FastifyPluginCallback = async (
+    fastify,
+    options,
+    next
+  ) => {
+    const sendResourceCooldowns = new BlockList(TRADE_COOLDOWN_MILLIS)
```

Comment:
Should be the same timeout for sending and for receiving trades, right?

```suggestion
    const sendResourceCooldowns = new BlockList(TRADE_DURATION_MILLIS)
```